test/pylib/scylla_cluster.py

```diff
@@ -989,6 +990,15 @@ async def handle_join_failure():
                           ip_addr, workdir, str(exc))
             await handle_join_failure()
             raise
+        except:
+            # catches everything
+            workdir = '<unknown>' if server is None else server.workdir.name
+            exc_type, exc_value, _ = sys.exc_info()
+            self.logger.error("Failed to start Scylla server at host %s in %s: %s(%s)",
+                              ip_addr, workdir, exc_type.__name__, exc_value)
```

Comment:
Why not use more modern `f"{}"`?

Reply:
just like the fmt library, Python logger [formats](https://docs.python.org/3/library/logging.html#logging.Logger.debug) the logging message lazily in the sense that the placeholder variables are formatted to their string representations only when the logger level is enabled. i'd prefer using this feature whenever possible, so that the codebase is more consistent in this regard. to avoid the the debate on why we use f-string here and here while use logger's builting formatting there and there.